examples/abac_rule_deny_model.conf

```diff
@@ -0,0 +1,11 @@
+[request_definition]
+r = sub, obj, act
+
+[policy_definition]
+p = sub_rule, obj, act, eft
+
+[policy_effect]
+e = some(where (p.eft == allow))
```

Comment:
This is missing the critical deny policy effect. Could you change this to:
```
e = some(where (p.eft == allow)) && !some(where (p.eft == deny))
```

Reply:
(I think this issue is specific to the `allow-and-deny` policy effect.)

Reply:
@Abingcbc

Reply:
Done❤️